src/test/java/apicurito/tests/steps/CommonSteps.java

```diff
@@ -109,40 +109,37 @@ public void deployAnotherCustomResource() {
         String cr =
             "https://gist.githubusercontent.com/mmajerni/e47e14f2a1c2bf934219cb3d4508e81c/raw/ff59f25b5a37918f19c69d70931154c081b683fa" +
                 "/operatorUpdateTest.yaml";
-        ApicuritoTemplate.deployCr(cr);
-
-        //Wait for Rollout until there is no unavailable pod
-        Integer tmp = Integer.MAX_VALUE;
-        while (tmp != null) {
-            //Wait for 5 seconds
-            try {
-                Thread.sleep(5000L);
-            } catch (InterruptedException e) {
-                e.printStackTrace();
-            }
-            tmp = OpenShiftUtils.getInstance().apps().deployments().inNamespace(TestConfiguration.openShiftNamespace()).list().getItems().get(1)
-                .getStatus().getUnavailableReplicas();
-        }
 
-        //Wait another 15 seconds because of termination running pods
-        try {
-            Thread.sleep(15000L);
-        } catch (InterruptedException e) {
-            e.printStackTrace();
-        }
+        ApicuritoTemplate.applyInOCP("Custom Resource", TestConfiguration.openShiftNamespace(), cr);
+        CommonUtils.waitForRollout();
     }
 
-    @When("check that apicurito image is {string}")
-    public void checkThatApicuritoImageIs(String image) {
+    @When("deploy another operator")
+    public void deployAnotherOperator() {
+        String operator =
+            "https://gist.githubusercontent.com/mmajerni/e7a4b5287f92c7ef228ba655883048be/raw/2eaaf61b377cf012dbf5c9bd4c8bb21447b6461f" +
+                "/apicuritoOperatorUpdate.yaml";
```

Comment:
Shouldn't the operator URL be provided as a env var?

Reply:
No Apicurito-UI image should. The operator is specified in the operator.yaml file

Reply:
I mean the above used: 
`        String operator =
            "https://gist.githubusercontent.com/mmajerni/e7a4b5287f92c7ef228ba655883048be/raw/2eaaf61b377cf012dbf5c9bd4c8bb21447b6461f" +
                "/apicuritoOperatorUpdate.yaml";`

Reply:
It should be placed in `TestConfiguration`, as all the other URL's

Reply:
It will be a static file so more likely I will create it as a file. Currently operator update test is not ready.